### ISSUE TITLE
`Mode.MARKDOWN` does not exist on type `Mode`

### ISSUE THREAD

**Issue by aminimalanimal** [hmsk/vite-plugin-markdown#417]
Version: 2.1.0

The documentation states:
> Enum for mode is provided as Mode
>
> import { Mode } from 'vite-plugin-markdown'
>
> console.log(Mode.HTML) //=> 'html'
> console.log(Mode.MARKDOWN) //=> 'markdown'

However, `Mode.MARKDOWN` is not defined. (All other documented modes are, however).

```ts
import { defineConfig } from 'vite'
import { plugin as mdPlugin, Mode } from 'vite-plugin-markdown'

export default defineConfig({
  plugins: [
    vue(),
    mdPlugin({
      mode: [Mode.MARKDOWN], // error: Property 'MARKDOWN' does not exist on type 'typeof Mode'
    }),
  ],
```

Consequently, I cannot import markdown as markdown in my JS.

```js
import { markdown as readme } from './README.md'
```

> ./README.md does not provide an export named 'markdown'

**Comment by fatcarter** [comment]
I have the same needs.
I see that the changes have been merged into the trunk, release the new version as soon as possible, please

**Comment by hmsk** [comment]
It's available from `2.2.0-2` on [beta tag](https://www.npmjs.com/package/vite-plugin-markdown/v/2.2.0-2) but it's not released on `latest` tag yet.
https://github.com/hmsk/vite-plugin-markdown/releases/tag/v2.2.0-2
